nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -161,12 +161,12 @@ def __init__(self, model, config_list, optimizer=None):
                 setattr(module, "weight_qmin", weight_q_min)
             if "input" in config.get("quant_types", []):
                 all_observers[layer_name]["input"] = default_histogram_observer()
-                setattr(module, "input_qmax", activation_q_max)
-                setattr(module, "input_qmin", activation_q_min)
+                setattr(module, "input_qmax", output_q_max)
```

Comment:
I think that we can unify `qmax` and `q_max`

Reply:
Good point, have unified them to `qmax`.